### PR TITLE
Improve documentation on `ThetaIntersection`

### PR DESCRIPTION
**Pull request by paulgb**
The documentation of ThetaIntersection was copied from ThetaUnion, I've updated it with ThetaIntersection-specific content.

This PR is also a trace bullet to see if you're currently open to accepting PRs, as I may have some other changes down the line if so.

### CODE REVIEW

File: src/wrapper/theta.rs [hunk 1]
```diff
@@ -133,8 +133,11 @@ pub struct ThetaIntersection {
 }
 
 impl ThetaIntersection {
-    /// Create a theta union over nothing, which corresponds to the
-    /// empty set.
+    /// Create a theta intersection over nothing, which corresponds to
+    /// an infinite set.
```
Comment (vlad17) [inline src/wrapper/theta.rs:137]: "a universal set containing all items, represented as None in `.sketch()`" (not all infinite sets have every key)

File: src/wrapper/theta.rs [hunk 2]
```diff
@@ -133,8 +133,11 @@ pub struct ThetaIntersection {
...
+    ///
+    /// Calls to `.sketch()` will return `None` unless `.merge()` has
```
Comment (vlad17) [inline src/wrapper/theta.rs:139]: Did you observe this behavior? Presumably, merging two universal sets which have no keys should yield another universal set still.

What matters is that `update()` has been called (or we merged with something updated)

**Review by vlad17 — CHANGES_REQUESTED** [review]

**Comment by vlad17** [comment]
> This PR is also a trace bullet to see if you're currently open to accepting PRs, as I may have some other changes down the line if so.

Definitely, thank you for the documentation update. Feel free to open issues on anything you'd like to be here, too.

File: src/wrapper/theta.rs (hunk 2 above)
Comment (paulgb) [reply src/wrapper/theta.rs:139]: The behavior I observed was that without calling `merge` at all (i.e. the intersection of “an empty set of sets”), `sketch` returned `None`. I didn't have a use case for this, I was just curious what would happen for the documentation.

**Comment by paulgb** [comment]
> > This PR is also a trace bullet to see if you're currently open to accepting PRs, as I may have some other changes down the line if so.
> 
> Definitely, thank you for the documentation update. Feel free to open issues on anything you'd like to be here, too.

Thanks, noted!

**Review by vlad17 — APPROVED** [review]

**Pull request merged**